### PR TITLE
Fix a typo in _index.md

### PR DESCRIPTION
**Pull request by trungnguyen**
missing 's' in the word 'public_server' in the code below data.example.public_server[s]

### CODE REVIEW

File: docs/content/_index.md [hunk 1]
```diff
@@ -1002,7 +1002,7 @@ opa run example.rego repl.input:input.json
 ```
 
 ```ruby
-> data.example.public_server[s]
+> data.example.public_servers[s]
```
Comment (srenatus) [inline docs/content/_index.md:1005]: The rule is defined as `public_server` here, though: https://github.com/open-policy-agent/opa/pull/6491/files#diff-5d9ba8bc1a568741b44ccc852748ca739d71f89dc1ac05a9f8d3787a064a549dR907

So either we keep it as-is, or change both locations (and perhaps others, if there are any).

**Review by srenatus — COMMENTED** [review]
Thanks for helping make the docs better! I think we'll need to do more here though 😅

**Comment by ashutosh-narkar** [comment]
@trungnguyen thanks for working on this. If you can address Stephan's comments above that would be great. Thanks!

File: docs/content/_index.md [hunk 2]
```diff
@@ -904,7 +904,7 @@ violation contains server.id if {                   # a server is in the violati
     server.protocols[_] == "telnet"                 # it contains the "telnet" protocol.
 }
 
-public_server contains server if {                  # a server exists in the public_server set if...
+public_servers contains server if {                  # a server exists in the public_servers set if...
```
Comment (ashutosh-narkar) [inline docs/content/_index.md:907]: This is called earlier in the policy. So we would need to update that too.

**Comment by ashutosh-narkar** [comment]
Thanks for the changes @trungnguyen! Can you please squash your commits and sign them off and we can get this in. See [this](https://www.openpolicyagent.org/docs/latest/contrib-code/#commit-messages) reference for authoring a good commit message.

File: docs/content/_index.md [hunk 3]
```diff
@@ -904,7 +904,7 @@ violation contains server.id if {                   # a server is in the violati
     server.protocols[_] == "telnet"                 # it contains the "telnet" protocol.
 }
 
-public_server contains server if {                  # a server exists in the public_server set if...
+public_servers contains server if {                  # a server exists in the 'public_servers' set if...
```
Comment (ashutosh-narkar) [inline docs/content/_index.md:907]: I think you missed one spot where the rule is called (`data.example.public_server[s]`). I think you had that in your original change.

Comment (trungnguyen) [reply docs/content/_index.md:907]: You are right, sorry. Will amend the last commit shortly.

Comment (ashutosh-narkar) [reply docs/content/_index.md:907]: Thanks! Please squash your commits and sign off when you're done.

**Review by ashutosh-narkar — APPROVED** [review]

**Pull request merged**